src/bap_mc/bap_mc.ml

```diff
@@ -64,6 +64,13 @@ let print_kinds insn =
   List.map ~f:sexp_of_kind |>
   List.iter ~f:(printf "%a@." Sexp.pp)
 
+let print_insn_oplen oplen_bases mem =
+  let len = (Memory.length mem) in
+  (List.iter oplen_bases
+	     ~f:(function
+		  | `hex -> printf (format_of_string "%x\n") len
```

Comment:
I will make the change.

At times, I have to use parenthesis to make sure that it passes the compiler, but then don't know how to realize the minimum number in OCaml.